zebra-state/src/service/queued_blocks.rs

```diff
@@ -15,13 +15,13 @@ use crate::{BoxError, CheckpointVerifiedBlock, SemanticallyVerifiedBlock};
 #[cfg(test)]
 mod tests;
 
-/// A finalized state queue block, and its corresponding [`Result`] channel.
+/// A checkpoint verified block queue, and its corresponding [`Result`] channel.
```

Comment:
This is just a single block

```suggestion
/// A queued checkpoint verified block, and its corresponding [`Result`] channel.
```